### ISSUE TITLE
Ares 64 - Quake II - Rendering issue with water

### ISSUE THREAD

**Issue by Mad-Matty** [TASEmulators/BizHawk#4037]
### Summary

Game stops rendering when jumping into water later in the game. The game it self is still going (no crash) so once you get out of the water, the game starts rendering again.  The first level is fine. its later on when the issue occurs.  It can be game breaking as you cant see where you are going collecting secrets underwater or navigating the sewer pipe in this level.

Quake II (USA)
SHA1:2EDB00E602E7C2813A5D6A04DFEC80487627237A

The issue does not occur in Ares v139

### Repro

1.Start from a password (Start/Newgame/Load/NoPak)
2. J6?0BT5MNRZ2QXLL (Sewer level - hard) (Not tested other difficulties)
3. Play through to the second room killing everything
4.  Jump into the water behind the pipe you see as you come through the door 
5. Experience gfx freeze (game is still running)

### Host env.

Bizhawk Dev build 2.9.2 Commit a1d4c5e, Windows 11 23H2, Radeon 7800XT, Ryzen 5950X

**Comment by YoshiRulz** [comment]
Did you mean to close this?

**Comment by Mad-Matty** [comment]
yes, it was a silly mistake my side. I had Skip VI Processing enabled for extra performance . Usually this just results in removing the excessive blurring n64 games have but for quake disabling caused other issues I didn't expect and jumped to conclusions.
